ERCS/eip-equity_token.md

```diff
@@ -0,0 +1,595 @@
+---
```

Comment:
```suggestion
---
eip: 7752
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.